From 787e06653c496d649930f122269b36e7f8a625b0 Mon Sep 17 00:00:00 2001
From: Chunhe Lan <Chunhe.Lan@freescale.com>
Date: Thu, 16 May 2013 16:45:53 +0800
Subject: [PATCH 337/547] powerpc/85xx: Update eeprom node and add rtc node in
 p1023rdb.dts

On the P1023RDB board, use 24c04 chip of eeprom with 0x53
address, and use mcp7941x chip of rtc with 0x6f address.

Signed-off-by: Chunhe Lan <Chunhe.Lan@freescale.com>
Change-Id: Ic4adf77fcf4774880e62a1c8bfe4a10fde1ebfd7
Reviewed-on: http://git.am.freescale.net:8181/2540
Reviewed-by: Zang Tiefei-R61911 <tie-fei.zang@freescale.com>
Tested-by: Review Code-CDREVIEW <CDREVIEW@freescale.com>
Reviewed-by: Fleming Andrew-AFLEMING <AFLEMING@freescale.com>
Tested-by: Fleming Andrew-AFLEMING <AFLEMING@freescale.com>
[Original patch taken from QorIQ-SDK-V1.4-SOURCE-20130625-yocto.iso]
Signed-off-by: Bin Jiang <bin.jiang@windriver.com>
---
 arch/powerpc/boot/dts/p1023rdb.dts | 11 ++++++++---
 1 file changed, 8 insertions(+), 3 deletions(-)

diff --git a/arch/powerpc/boot/dts/p1023rdb.dts b/arch/powerpc/boot/dts/p1023rdb.dts
index f2758b8..24e71f8 100644
--- a/arch/powerpc/boot/dts/p1023rdb.dts
+++ b/arch/powerpc/boot/dts/p1023rdb.dts
@@ -65,9 +65,14 @@
 		ranges = <0x0 0x0 0xff600000 0x200000>;
 
 		i2c@3000 {
-			at24@50 {
-				compatible = "at24,24c08";
-				reg = <0x50>;
+			eeprom@53 {
+				compatible = "at24,24c04";
+				reg = <0x53>;
+			};
+
+			rtc@6f {
+				compatible = "microchip,mcp7941x";
+				reg = <0x6f>;
 			};
 		};
 
-- 
1.8.4.93.g57e4c17

